app/controllers/comment_controller.php

```diff
@@ -0,0 +1,69 @@
+<?php
+class CommentController extends AppController
+{
+    public function edit()
+    {
+        $comment = new Comment;
+        $page = Param::get('page_next', 'edit');
+        $comment_id = Param::get('id');
+        $comments = Comment::getCommentsInfo($comment_id);
+        $error = false;
+
+        switch ($page) {
+            case 'edit':
```

Comment:
please use constant variables rather than magic values.. 
```php
const PAGE_EDIT = 'edit';

...

switch ($page) {
    case self::PAGE_EDIT:
        //do process
}
```